### ISSUE TITLE
Display an information text on the map in the mobile view for posts without geolocation data

### ISSUE THREAD

**Issue by MaiThiD** [hochschule-darmstadt/einander-helfen#243]
**Reason (Why?)**
Due to the transmission of empty geolocation data, the map button of the affected posts is disabled in the detail view. In the mobile view the detail and map button are not visible anymore. If you select a post, the pin of the selected post will be automatically zoomed in, in the map. If you select a post which has no geolocation, nothing will happen. It is neccessary that the user get some information why no pin is zoomed in.

**Solution (What?)**
If the user selects a post without geolocation data, a transparent black background should be displayed over the map which contains an information text why the pin is not zoomed in. The text should be something like 'Für den ausgewählten Post gibt es keinen Pin auf der Karte'. After deselecting the post, the background and the text should disappear. This should also happend, if he/she selects another post which has geolocation data. The text should be white. 
 
**Out of scope**
Desktop version

**Relation to other Issues**
#240 #122 #215 

**Acceptance criteria**
The (white) text and the transparent black background should be only displayed for posts without geolocation data.
This should look like the current "Verwende Strg+Scrollen zum Zoom der Karte"

![image](https://user-images.githubusercontent.com/64202398/102495287-34239d80-4076-11eb-82e0-fe46d283f8cc.png)

**Additional text**
Issue #76 should be merged first before this can be implemented.
